pkg/plan/direct.go

```diff
@@ -23,18 +23,19 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/cectc/dbpack/pkg/constant"
+	"github.com/cectc/dbpack/pkg/log"
 	"github.com/cectc/dbpack/pkg/proto"
 	"github.com/cectc/dbpack/third_party/parser/ast"
 	"github.com/cectc/dbpack/third_party/parser/format"
 )
 
-type ShowTableMetaPlan struct {
-	Stmt     *ast.ShowStmt
+type DirectlyQueryPlan struct {
```

Comment:
`Directly` is an adverb, QueryPlan is a noun, so it's better to use `DirectQueryPlan`